packages/styles/tag-button.css

```diff
@@ -0,0 +1,8 @@
+.TagButton {
+  vertical-align: middle;
+}
+
+.TagButton__icon svg {
+  width: 15px;
+  height: 15px;
+}
```

Comment:
I'm assuming these sizes came from @dequelabs/cauldron-design? Unfortunately I don't see it listed on the original issue/uxpin but if these are the right sizes, they need to be set via css vars:

```css
:root {
  --tag-button-icon-size: ...
}
```

Reply:
I'm wrong, sorry i get it from Monitor, but now i see that real design even on the monitor has 10px..